src/features/tab-view/hooks/use-table-export.tsx

```diff
@@ -5,8 +5,13 @@ import { notifications } from '@mantine/notifications';
 import { copyToClipboard } from '@utils/clipboard';
 import { escapeCSVField } from '@utils/helpers';
 import { stringifyTypedValue } from '@utils/db';
+import { TabId } from '@models/tab';
+import { useTabNameMap } from '@store/app-store';
+
+export const useTableExport = (dataAdapter: DataAdapterApi, tabId: TabId) => {
+  const tabNameMap = useTabNameMap();
```

Comment:
this will cause unnecessary unrelated re-renders when any tab (not only this one) will change name or we add new ones. You can just read this single name from state
```suggestion
  const tabName = useAppStore((state) => state.tabs.get(tabId) || 'unknown-tab-export.csv');
```